packages/venia-ui/lib/RootComponents/Category/category.js

```diff
@@ -157,22 +157,25 @@ const Category = props => {
         return fullPageLoadingIndicator;
     }
 
+    const metaDescription =
+        data &&
+        data.category &&
+        data.category.meta_description &&
+        data.category.meta_description !== null
```

Comment:
I think this line is redundant— truthy `meta_description` can't be `null`.

Reply:
Thanks - removed in f13544cfe2c0df96eb5d0ea0294e20201ca877bd